src/lib/stores/parent-resource.store.ts

```diff
@@ -11,3 +12,30 @@ export const parentResourceNameToInfoMap = derived(parentResources, ($parentReso
         (r) => r[0]
     )
 );
+
+export const guideResources = derived(parentResources, ($parentResources) =>
+    $parentResources.filter((r) => r.resourceType === 'Guide')
+);
+
+const locallyStoredGuide: ApiParentResource | undefined = (() => {
+    const localGuideShortName = browser && localStorage.getItem('bibleWellCurrentGuide');
+    if (localGuideShortName) {
+        let foundResource: ApiParentResource | undefined;
+
+        guideResources.subscribe((guideResources) => {
+            foundResource = guideResources.find((r) => r.shortName === localGuideShortName);
+        });
```

Comment:
Nit here: if you use `get` from `svelte/store` then it will handle the subscription and unsubscription. As is we're technically leaving the subscription there after the code runs (not that it will actually matter since this should only run once).